oidc-service-impl/src/main/java/io/mosip/esignet/services/TokenServiceImpl.java

```diff
@@ -227,6 +229,7 @@ private JSONObject buildIDToken(String subject, String audience, int validitySec
         payload.put(NONCE, nonce == null ? transaction.getNonce() : nonce);
         List<String> acrs = authenticationContextClassRefUtil.getACRs(transaction.getProvidedAuthFactors());
         payload.put(ACR, String.join(SPACE, acrs));
+        payload.put(JTI, UUID.randomUUID().toString());
```

Comment:
Instead of only using UUID , lets use the "IdentityProviderUtil.createTransactionId()" method to generate unique ID. Previously we had noticed 2 different machines produce the same UUID when invoked at the same time.